oligo_designer_toolsuite/oligo_specificity_filter/_filter_blastn.py

```diff
@@ -214,16 +415,40 @@ class BlastNSeedregionFilterBase(BlastNFilter):
     :type blast_hit_parameters: dict
     :param dir_output: Directory for saving output files and BLAST databases.
     :type dir_output: str
+    :param names_search_output: Column names for parsing BLAST search output.
+    :type names_search_output: list
+    :param ai_filter: The machine learning model used to filter the oligos {None, 'hybridization_probability'}, defaults to None
+    :type ai_filter: str, optional
+    :param ai_filter_thershold: The threshold below which the oligos are filtered, defaults to None
+    :type ai_filter_thershold: float, optional
+    :param ai_filter_path: The path to the machine learning model used to filter the oligos, if None the pretrained model provided will be used, defaults to None
+    :type ai_filter_path: str, optional
     """
 
     def __init__(
         self,
         blast_search_parameters: dict = {},
         blast_hit_parameters: dict = {},
         dir_output: str = "output",
+        names_search_output: list = [
+            "query",
+            "reference",
+            "alignment_length",
+            "query_start",
+            "query_end",
+            "query_length",
+            "query_sequence",
+            "reference_start",
+            "reference_end",
+            "reference_sequence",
+            "reference_strand"
+        ],
+        ai_filter: str = None,
+        ai_filter_threshold: float = None,
+        ai_filter_path: str = None,
```

Comment:
same as above. move ai filters to own class and don't use all names_search_output per default but only keep minimum required ones for pure blast filter